service-api/app/src/App/src/DataAccess/DynamoDb/UserLpaActorMap.php

```diff
@@ -56,22 +51,34 @@ public function get(string $lpaActorToken): ?array
 
     /**
      * @inheritDoc
+     * @throws Exception
      */
-    public function create(string $lpaActorToken, string $userId, string $siriusUid, int $actorId)
-    {
-        // The current DateTime, including microseconds
-        $now = (new DateTime())->format('Y-m-d\TH:i:s.u\Z');
+    public function create(
+        string $lpaActorToken,
+        string $userId,
+        string $siriusUid,
+        string $actorId,
+        string $expiryInterval = null
+    ) {
+        $added = new \DateTime();
+        $array = [
+            'Id'        => ['S' => $lpaActorToken],
+            'UserId'    => ['S' => $userId],
+            'SiriusUid' => ['S' => $siriusUid],
+            'ActorId'   => ['N' => $actorId],
+            'Added'     => ['S' => $added->format('Y-m-d\TH:i:s.u\Z') ]
+        ];
+
+        //Add ActivateBy field to array if expiry interval is present
+        if ($expiryInterval != null) {
```

Comment:
```suggestion
        if ($expiryInterval !== null) {
```
Always be super explicit (granted, that in the case of null there isn't much that can go wrong)